reactDashboard/src/components/HeatMap/Heatmap.css

```diff
@@ -2,34 +2,58 @@
   overflow-y: auto;
   max-height: 600px;
   height: 100%;
-  background-color: #ffffff;
+  background-color: white;
+}
+
+.darkmode .heatmap {
+  background-color: rgb(64, 64, 64);
+  box-shadow: none;
+}
+
+.heatmap-container {
+  padding: 0;
 }
 
 .heatmap table {
-  overflow-y: auto;
+  border-collapse: separate;
+  border-spacing: 0;
 }
 
 .heatmap table th {
   vertical-align: top;
   position: sticky;
+  position: -webkit-sticky;
   top: 0;
-  z-index: 2;
   background-color: white;
   color: #133a53;
 }
 
-.heatmap table th .subheading {
+.darkmode .heatmap table th {
+  color: #f2f2f2;
+  background-color: rgb(64, 64, 64);
+}
+
+.heatmap table th div.subheading {
   font-weight: normal;
 }
 
+.table th {
```

Comment:
This selector (and the next one) is still too generic. Also, use the element table rather than the class .table. Finally, have the selector and its darkmode override match (one has thead)

Reply:
There is a block at line 31 that this block could be merged with